zbus/src/object_server/node.rs

```diff
@@ -119,11 +119,11 @@ impl Node {
         arc_iface: ArcInterface,
     ) -> bool {
         match self.interfaces.entry(name) {
-            Entry::Vacant(e) => {
+            btree_map::Entry::Vacant(e) => {
```

Comment:
This and the following hunk becomes unnecessary if you keep the import of the `Entry` instead of the module? 😉